jobs/elasticsearch_config/templates/bin/upload-config.erb

```diff
@@ -1,26 +1,17 @@
 #!/bin/bash
 
 set -ex
-
+export DATA_DIR=/var/vcap/data/elasticsearch_config
 <%
-  elasticsearch_host = nil
-  if_link("elasticsearch") { |elasticsearch_link| elasticsearch_host = elasticsearch_link.instances.first.address }
-  unless elasticsearch_host
-    elasticsearch_host = p("elasticsearch_config.elasticsearch.host")
-  end
-
-  elasticsearch_port = nil
-  if_link("elasticsearch") { |elasticsearch_link| elasticsearch_port = elasticsearch_link.p("elasticsearch.port") }
-  unless elasticsearch_port
-    elasticsearch_port = p("elasticsearch_config.elasticsearch.port")
-  end
+elasticsearch_host = p("elasticsearch_config.elasticsearch.host")
```

Comment:
This will also require updating the deployment, since we're getting this via links right now